### PR TITLE
Pattern Journey. Description

### LINKED ISSUE #4: Ex: Pattern Journey
- [ ] Description
- [ ] Design canvas

### PR DESCRIPTION
**Pull request by varya**
Closes #4

### CODE REVIEW

File: slides/ds101-worskhop.mdx [hunk 1]
```diff
@@ -195,6 +195,49 @@ to find out the relevant parts of your design system.
...
+1. Naming the 5 stages (15 minutes)
+1. "What" lane (45 minutes)
+1. "Who" lane (15 minutes)
+1. "How" lane (15 minutes)
+
+# Conclusion
+
+Every time when we run this exercise, we get very different results. But interpretation of
+these results was always fruitful. In this case, we analyse the outcome we get now and
+share information about pattern journeys at our projects.
```
Comment (okonet) [inline slides/ds101-worskhop.mdx:229]: Can you add slides showing how it was in your projects showing boards and going through interpretation?

Comment (varya) [reply slides/ds101-worskhop.mdx:229]: If you mean the workshop board itself, its NDA :-( The Pattern Journey is public, I was thinking to add slides about it and about your state machine in my next PR.

**Comment by varya** [comment]
@okonet Updated with more slides.

File: slides/ds101-worskhop.mdx [hunk 2]
```diff
@@ -238,6 +238,26 @@ share information about pattern journeys at our projects.
 
 ---
 
+<Primary># Interpretation</Primary>
```
Comment (okonet) [inline slides/ds101-worskhop.mdx]:
```suggestion
<Primary>

# Interpretation

</Primary>
```

**Review by okonet — CHANGES_REQUESTED** [review]
Otherwise this won't render correctly

**Review by okonet — APPROVED** [review]

**Review by okonet — APPROVED** [review]

**Pull request merged**